src/mito2/src/sst/file_purger.rs

```diff
@@ -55,25 +54,29 @@ impl LocalFilePurger {
 }
 
 impl FilePurger for LocalFilePurger {
-    fn send_request(&self, request: PurgeRequest) -> Result<()> {
+    fn send_request(&self, request: PurgeRequest) {
         let file_id = request.file_id;
         let region_id = request.region_id;
         let sst_layer = self.sst_layer.clone();
 
-        self.scheduler.schedule(Box::pin(async move {
-            sst_layer.delete_sst(file_id).await.map_err(|e| {
-                error!(e; "Failed to delete SST file, file: {}, region: {}", 
-                    file_id.as_parquet(), region_id);
+        let _ = self
+            .scheduler
+            .schedule(Box::pin(async move {
+                if let Err(e) = sst_layer.delete_sst(file_id).await {
+                    error!(e; "Failed to delete SST file, file: {}, region: {}", 
+                        file_id.as_parquet(), region_id);
+                } else {
+                    debug!(
```

Comment:
I'd prefer change to `info!` since deleting SST files is a relatively sensitive operation.